docs/REST API/rest-api.md

```diff
@@ -4160,22 +4160,24 @@ Get aggregate status counts of MDM profiles applying to macOS hosts enrolled to
 
 ### Run custom MDM command
 
-This endpoint tells Fleet to run a custom MDM command, on the targeted macOS hosts, the next time they come online.
+> `POST /api/v1/fleet/mdm/apple/enqueue` API endpoint is deprecated as of Fleet 4.40. It is maintained for backward compatibility. Please use the new API endpoint below. See old API endpoint docs [here](https://github.com/fleetdm/fleet/blob/ee02782eaf84c121256d73abc20b949d31bf2e57/docs/REST%20API/rest-api.md#run-custom-mdm-command).
 
-`POST /api/v1/fleet/mdm/apple/enqueue`
+This endpoint tells Fleet to run a custom MDM command, on the targeted macOS or Windows hosts, the next time they come online.
+
+`POST /api/v1/fleet/mdm/commands/run`
 
 #### Parameters
 
 | Name                      | Type   | In    | Description                                                               |
 | ------------------------- | ------ | ----- | ------------------------------------------------------------------------- |
-| command                   | string | json  | A base64-encoded MDM command as described in [Apple's documentation](https://developer.apple.com/documentation/devicemanagement/commands_and_queries). Supported formats are standard ([RFC 4648](https://www.rfc-editor.org/rfc/rfc4648.html)) and raw (unpadded) encoding ([RFC 4648 section 3.2](https://www.rfc-editor.org/rfc/rfc4648.html#section-3.2)) |
-| device_ids                | array  | json  | An array of host UUIDs enrolled in Fleet's MDM on which the command should run.                   |
+| command                   | string | json  | A base64-encoded MDM command as described in [Apple's documentation](https://developer.apple.com/documentation/devicemanagement/commands_and_queries) or [Windows's documentation](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-mdm/0353f3d6-dbe2-42b6-b8d5-50db9333bba4). Supported formats are standard ([RFC 4648](https://www.rfc-editor.org/rfc/rfc4648.html) aka. Main) and raw (unpadded) encoding ([RFC 4648 section 3.2](https://www.rfc-editor.org/rfc/rfc4648.html#section-3.2)). You can use the [online detector](https://base64.guru/tools/detector) to ensure you're using the right format. |
+| host_ids                | array  | json  | An array of host UUIDs enrolled in Fleet on which the command should run. |
```

Comment:
Looks like the API accepts host UUIDs which I think is different than host IDs? 

If this is the case, maybe `host_ids` isn't explicit enough.

I also realized we should be looping in @mna here since this parameter name change impacts the Windows MDM commands story (#13069) that's being worked on in the current sprint.

Reply:
@marko-lisica what do you think?

Reply:
I agree. Let's go w/ `host_uuids` and `host_uuid` (instead of `device_id` in the response).

@marko-lisica can you please make these changes to the PR to lock this in?

Heads up @mna.